src/Searchbar.vue

```diff
@@ -1,55 +1,84 @@
 <script>
 import typeahead from './Typeahead.vue';
+import searchbarPageItem from './SearchbarPageItem.vue';
 
 export default {
   extends: typeahead,
   computed: {
     primitiveData() {
-      function getTotalMatches(searchTarget, regexes) {
-        return regexes.reduce((total, regex) => (regex.test(searchTarget) ? total + 1 : total), 0);
+      // Returns the total number of matches between an array of regex patterns and string search targets.
+      function getTotalMatches(regexes, ...searchTargets) {
```

Comment:
Sorry if it wasn't clear before, but I was thinking of having a variable `searchTargets`. I don't feel that the spread operator gives us much benefit here, and makes it a little more confusing at the call site.

Reply:
Agreed, I'm still not so sure what you mean by "having a variable searchTargets" though;

As a guess, do you mean removing `searchTarget/searchTargets` as a parameter from `getTotalMatches` altogether, and using a variable local to `primitiveData()`?

Reply:
I was thinking something along these lines:
```
function getTotalMatches(regexes, searchTargets) {
  ...
}

const searchTargets = [...];
const totalMatches = getTotalMatches(regexes, searchTargets);
```